libraries/rib-base/src/main/java/com/badoo/ribs/android/dialog/DialogLauncher.kt

```diff
@@ -1,8 +1,10 @@
 package com.badoo.ribs.android.dialog
 
+import androidx.appcompat.app.AlertDialog
+
 interface DialogLauncher {
 
-    fun show(dialog: Dialog<*>, onClose: () -> Unit)
+    fun show(dialog: Dialog<*>, onClose: () -> Unit, onShown: (AlertDialog) -> Unit)
```

Comment:
Reorder + rename for consistency?

```suggestion
    fun show(dialog: Dialog<*>, onShow: (AlertDialog) -> Unit, onClose: () -> Unit)
```

Reply:
We should be **very** careful with migrations then.
If we have `show(dialog) { someCode() }` then it will change current behaviour.